src/templates/quote/icon.js

```diff
@@ -0,0 +1,36 @@
+import { React } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * @param {object} props
+ *   Props.
+ * @param  {string} props.stroke
+ *   The stroke color.
+ * @param {number} props.strokeWidth
+ *   The stroke width.
+ * @param {string} props.className
+ *   The class name.
+ * @returns {JSX.Element}
+ *   The component.
+ */
+function Icon({ stroke, strokeWidth, className }) {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" width="38" height="28" className={className} viewBox="0 0 38 28">
```

Comment:
I have seen examples where svg is in svg files an using import to get content into an const. Would that not make editing these svg's later on easier?